### DISCUSSION TITLE
Changing theme of syntax highlighting

### DISCUSSION THREAD

**Discussion by AlTosterino** [cotes2020/jekyll-theme-chirpy#1392] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Hey!

Is there a easy way to customize syntax highlighting theme from rouge?
For example, running: `rougify help style` outputs available themes, but is there a way to easily override theme for Chirpy?

### Operations you have already tried

I went through Chirpy source code and I found `syntax-light.css syntax-dark.css` but I don't know how to correctly swap them 🤔

**Comment by tis-starlight** [comment]
See discussion #2298
